virttest/rss_client.py

```diff
@@ -135,6 +135,8 @@ def close(self):
         self._socket.close()
 
     def _send(self, sr, timeout=60):
+        if isinstance(sr, six.string_types):
```

Comment:
Maybe we can use existed astring.is_text to do conditional judgement first ,then do encode().